torch/nn/modules/activation.py

```diff
@@ -1133,28 +1152,29 @@ def forward(
             # generator expressions.
             if torch.overrides.has_torch_function(tensor_args):
                 why_not_fast_path = "some Tensor argument has_torch_function"
-            elif not all([(x is None or x.is_cuda or 'cpu' in str(x.device)) for x in tensor_args]):
+            elif not all([_arg_cuda_or_cpu(x) for x in tensor_args]):
```

Comment:
here square brackets are not strictly needed...

Reply:
actually, for Torchscript they are needed because torchscript only supports a subset of Python features :(